diff -up fonttosfnt-1.0.5/Makefile.am.1~ fonttosfnt-1.0.5/Makefile.am
--- fonttosfnt-1.0.5/Makefile.am.1~	2018-09-11 17:06:29.100061153 +0200
+++ fonttosfnt-1.0.5/Makefile.am	2018-09-11 17:07:11.857592819 +0200
@@ -23,7 +23,7 @@ SUBDIRS = man
 bin_PROGRAMS = fonttosfnt
 
 AM_CFLAGS = -DXVENDORNAME=\"X.Org\ Foundation\" $(FONTTOSFNT_CFLAGS) -DXVENDORNAMESHORT=\"X.Org\" -D_DEFAULT_SOURCE -D_GNU_SOURCE $(CWARNFLAGS)
-fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm
+fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm -lbz2
 
 fonttosfnt_SOURCES =	\
         fonttosfnt.c \
diff -up fonttosfnt-1.0.5/Makefile.in.1~ fonttosfnt-1.0.5/Makefile.in
--- fonttosfnt-1.0.5/Makefile.in.1~	2018-09-11 17:06:29.100061153 +0200
+++ fonttosfnt-1.0.5/Makefile.in	2018-09-11 17:07:20.228501098 +0200
@@ -378,7 +378,7 @@ SUBDIRS = man
 AM_CFLAGS = -DXVENDORNAME=\"X.Org\ Foundation\" $(FONTTOSFNT_CFLAGS) \
 	-DXVENDORNAMESHORT=\"X.Org\" -D_DEFAULT_SOURCE -D_GNU_SOURCE \
 	$(CWARNFLAGS) $(am__append_2)
-fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm
+fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm -lbz2
 fonttosfnt_SOURCES = fonttosfnt.c fonttosfnt.h read.c struct.c util.c \
 	write.c $(am__append_1)
 MAINTAINERCLEANFILES = ChangeLog INSTALL
