diff --git a/Makefile.am b/Makefile.am
index 3fe03df..a0735f1 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -22,7 +22,7 @@
 bin_PROGRAMS = fonttosfnt
 
 AM_CFLAGS = -DXVENDORNAME=\"X.Org\ Foundation\" $(FONTTOSFNT_CFLAGS) -DXVENDORNAMESHORT=\"X.Org\" -D_BSD_SOURCE
-fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm
+fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm -lbz2
 
 fonttosfnt_SOURCES =	\
         fonttosfnt.c \
diff --git a/Makefile.in b/Makefile.in
index 6d6c340..4d5a4d4 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -223,7 +223,7 @@ top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CFLAGS = -DXVENDORNAME=\"X.Org\ Foundation\" $(FONTTOSFNT_CFLAGS) \
 	-DXVENDORNAMESHORT=\"X.Org\" -D_BSD_SOURCE $(am__append_2)
-fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm
+fonttosfnt_LDADD = $(FONTTOSFNT_LIBS) -lm -lbz2
 fonttosfnt_SOURCES = fonttosfnt.c fonttosfnt.h read.c struct.c util.c \
 	write.c $(am__append_1)
 appman_PRE = \
